app.py

```diff
@@ -1,27 +1,34 @@
-from flask import Flask, redirect, render_template
+from flask import Flask, redirect, render_template, session, request
 
 from src.repositories.movie_repository import get_movie_repository
 
 app = Flask(__name__)
 
+
+
+
+
 # Get the movie repository singleton to use throughout the application
 movie_repository = get_movie_repository()
 
-
 @app.get('/')
 def index():
     return render_template('index.html')
 
-
+#Jaidens List All Movies Function
 @app.get('/movies')
 def list_all_movies():
-    # TODO: Feature 1
-    return render_template('list_all_movies.html', list_movies_active=True)
+    movies = []
```

Comment:
Line 21 movies = [] will probably need to be removed to prevent duplicate lists since Cindy had it in her code outside the functions. The following lines should still work when we merge everything.